File: python/7 kyu/209.py
# https://www.codewars.com/kata/539ee3b6757843632d00026b/train/python

# Passed

def capitals(word):
    capital_indices = []
    for idx, letter in enumerate(word):
        if letter.isupper():
            capital_indices.append(idx)
            
    return capital_indices

output = capitals('CodEWaRs')
print(output)

File: nodejs/7 kyu/209.js
// https://www.codewars.com/kata/56dbed3a13c2f61ae3000bcd/train/javascript

// Passed

function noonerize(numbers) {
  if (!numbers.every((num) => typeof num == "number")) return "invalid array";
  let numbersAsString = numbers.map((num) => String(num));
  let [firstNumString, secondNumString] = numbersAsString;
  let [modifiedFirstNum, modifiedSecondNum] = [
    secondNumString[0] + firstNumString.slice(1),
    firstNumString[0] + secondNumString.slice(1),
  ];
  let AbsoluteDifference = Math.abs(
    Number(modifiedFirstNum) - Number(modifiedSecondNum)
  );
  return AbsoluteDifference;
}

const output = noonerize([55, 63]);
console.log(output);